Rename and improve sm getstarted and install section (#5895)
camunda/camunda-docs: docs/self-managed/setup/overview.md

---
id: overview

> title: "Install Camunda 8 Self-Managed in production environments"

Author: Good thought! I adjusted the text! Better?

- [**Docker**](./deploy/other/docker.md): All Camunda components are available as [Docker images](https://hub.docker.com/u/camunda) and can be used in production on Linux systems. Windows or macOS are only supported for development environments, not for production.
- [**Manual**](./deploy/local/manual.md): Run each Java application manually on virtual machines or bare-metal servers with a supported Java Virtual Machine (JVM). This method offers a significant amount of flexibility but requires manual configuration of component interactions. It is not recommended unless necessary, as you will need to configure the details for the components to interact correctly yourself. Windows or macOS are only supported for development environments, not for production.

:::info

> If you're looking to run Camunda 8 in a local environment for pure development or evaluation purposes, see [**Run locally**](../run-locally/index.md)

Reviewer: ```suggestion
To run Camunda 8 in a local environment for development or evaluation purposes only, review [running locally](../run-locally/index.md)
```
